hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOzoneManagerHAWithStoppedNodes.java

```diff
@@ -474,7 +474,7 @@ void testIncrementalWaitTimeWithSameNodeFailover() throws Exception {
     String leaderOMNodeId = omFailoverProxyProvider.getCurrentProxyOMNodeId();
 
     getCluster().stopOzoneManager(leaderOMNodeId);
-    Thread.sleep(NODE_FAILURE_TIMEOUT * 4);
+    getCluster().waitForLeaderOM();
     createKeyTest(true); // failover should happen to new node
```

Comment:
Seems like the exception may still happen:

```
	at org.apache.hadoop.ozone.om.TestOzoneManagerHA.createKeyTest(TestOzoneManagerHA.java:409)
	at org.apache.hadoop.ozone.om.TestOzoneManagerHAWithStoppedNodes.testIncrementalWaitTimeWithSameNodeFailover(TestOzoneManagerHAWithStoppedNodes.java:478)
```

https://github.com/raju-balpande/apache_ozone/actions/runs/8878560802/job/24374580998#step:6:22713

Reply:
Oh, I wrongly specified the initial check which I made on master branch... We can refer to following.
https://github.com/raju-balpande/apache_ozone/actions/runs/8921923082 .. correcting it in description as well.

Reply:
That looks much better, thanks. :)